### DISCUSSION TITLE
Failed to run simulation

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/FEPX#76] [Q&A] [answered]
version：neper 4.3.1；fepx 1.3.0
problem：
When running the simulation similar to example 1, it is found that if the Morpho of the created rve is "gg", it can run normally,
 but if it is changed to Voronoi tessellations, an error will appear. The instructions I use are as follows：

neper -T -n 49 -domain 'cube(7,18,7)' -o simulation
neper -M simulation.tess -rcl 0.4 -order 2

[simulation.config.txt](https://github.com/acmelab-ua/FEPX/files/8254338/simulation.config.txt)

mpirun -np 4 fepx

the following error messages will appear:
mpirun noticed that process rank 3 with PID 0 on node ubuntu exited on signal 9 (Killed).

so I want to know whether the above simulation is correct and can run normally on your computer？Thanks.

**Comment by mkasemer** [comment, marked as answer]
@Ming-is - Apologies for the delay in response, my university was on break.

I am unable to reproduce the error on my end. I am using Neper 4.4.1 and FEPX 1.3.0. There is seemingly nothing wrong with the Neper commands that you give, and the supplied FEPX config file contains no immediately apparent errors. I am able to run a simulation using those commands, the supplied FEPX config file, executed using mpirun with 4 processors.

The error you detail is likely something with the MPI installation on your system. This could be any number of things, and I am not an expert on MPI. The first thing that comes to mind is that there could be a mismatch between the compiler you used to build FEPX and the compiler used to build the MPI instance on the system, but I am uncertain if it would cause this error specifically.

**Reply by Ming-is** [reply]
When I delete  ”-rcl 0.4“  here, the simulation can run again. It's strange.

**Reply by rquey** [reply]
What I'm going to suggest does not have to do with MPI, but, since the `gg` tessellation runs, I would recommend to add `-reg 1` to your `-T` command. Standard Voronoi tessellations are much more subjected to generating ill-shaped elements than `gg` tessellations, which can be avoided using `-reg 1`. I think this might be what the problem is... or at least it would be good practice.

**Reply by mkasemer** [reply]
I agree that regularization should always be performed, but the error is strange and seems unrelated (perhaps not).
